internal/driver/glfw/window.go

```diff
@@ -423,10 +424,11 @@ func (w *window) Hide() {
 }
 
 func (w *window) Close() {
-	if w.viewport == nil {
+	if w.closing {
 		return
 	}
 
+	w.closing = true
 	w.viewport.SetShouldClose(true)
```

Comment:
Should this still be protected by a nil check?
```suggestion
	if v := w.viewport; v != nil {
		v.SetShouldClose(true)
	}
```

Reply:
I don't think so - previously viewport was set to nil to show it was closed, now it is only nil before the window appears.
I guess it couldn't hurt to use the suggested helper at the top instead though.